head/firmware/motor_hardware_common.c

```diff
@@ -231,14 +231,21 @@ void MX_GPIO_Init(void) {
     initialize_rev_specific_pins();
 }
 
+#if PCBA_PRIMARY_REVISION == 'b' || PCBA_PRIMARY_REVISION == 'a'
+    static const int encoder_direction = TIM_ICPOLARITY_RISING;
```

Comment:
The direction is flipped for both motors?

Reply:
yeah, both motors need the change right?

Reply:
since they both changed the number of gears